hack/import-restrictions.yaml

```diff
@@ -142,23 +149,29 @@
 - baseImportPath: "./vendor/k8s.io/kube-controller-manager/"
   allowedImports:
   - k8s.io/apimachinery
-  - k8s.io/apiserver
+  - k8s.io/component-base
+  - k8s.io/klog
   - k8s.io/utils
 
 - baseImportPath: "./vendor/k8s.io/kube-proxy/"
   allowedImports:
   - k8s.io/apimachinery
+  - k8s.io/component-base
+  - k8s.io/klog
+  - k8s.io/utils
```

Comment:
are these non-staging repos necessary here? As it was working before, I would gues the answer is no.